source/Planet.cpp

```diff
@@ -472,10 +472,22 @@ void Planet::Bribe(bool fullAccess) const
 // Demand tribute, and get the planet's response.
 string Planet::DemandTribute(PlayerInfo &player) const
 {
-	if(player.GetCondition("tribute: " + name))
-		return "We are already paying you as much as we can afford.";
+	static const int TRIBUTE_IGNORED = 1;
```

Comment:
I don't know enough to be sure, but should these be defined in `namespace` at the top of the file instead? Arguably they should also be `static constexpr`, but I don't think this is generally used in the code right now.